packages/tradershub/src/features/cfd/modals/TradeModal/TradeModal.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC, useEffect } from 'react';
+import QRCode from 'qrcode.react';
+import { Provider } from '@deriv/library';
+import { Text, useBreakpoint } from '@deriv/quill-design';
+import { Modal } from '../../../../components/Modal';
+import { THooks, TMarketTypes, TPlatforms } from '../../../../types';
+import { AppToIconMapper, LinksMapper, PlatformDetails, TAppLinks } from '../../constants';
+import { MT5TradeScreen } from '../../screens/MT5TradeScreen';
+
+type TTradeModalProps = {
+    account?: THooks.CtraderAccountsList | THooks.DxtradeAccountsList | THooks.MT5AccountsList;
+    marketType?: TMarketTypes.All;
+    platform: TPlatforms.All;
+};
+
+const TradeModal: FC<TTradeModalProps> = ({ account, marketType, platform }) => {
+    const { isDesktop } = useBreakpoint();
+    const { setCfdState } = Provider.useCFDContext();
+
+    useEffect(() => {
+        setCfdState('marketType', marketType);
+        setCfdState('platform', platform);
+        if (platform === 'mt5') setCfdState('accountId', (account as THooks.MT5AccountsList)?.loginid);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    const appOrder = ['ios', 'android', 'huawei'];
+
+    return (
+        <Modal>
+            <Modal.Header title='Trade' />
+            <Modal.Content>
+                <MT5TradeScreen account={account} />
+            </Modal.Content>
+            <Modal.Footer>
+                <div className='pt-50 min-h-[19rem] lg:pt-[inherit] lg:min-h-[inherit] flex justify-center items-center flex-col h-fit w-full gap-800'>
```

Comment:
what does` [inherit]` do? Is there any difference having it if not can we remove it? I'm afraid it will abit confusing for the others.